### PR TITLE
Output appropriately sized images

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
i.e. images in the 'related' box don't need to be 700px wide, and response-ify all the images (with `srcset`/`sizes`)

Also move the encoding of the url into the `figure-variant` transform

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.5",
   "main": "main.js",
   "dependencies": {
-    "cheerio": "^0.19.0",
+    "cheerio": "^0.20.0",
```
Comment (andygnewman) [inline package.json]: We had memory issues the last time we upgraded to 0.20, hence we went back to 0.19. Here's the issue @wheresrhys  raised https://github.com/cheeriojs/cheerio/issues/856

Comment (ironsidevsquincy) [inline package.json]: Ohh, thanks. Darn, means I can't use the `wrap` method

Comment (andygnewman) [inline package.json]: Hadn't seen the wrap method before - looks useful, if only they sorted out the memory issue.

**Comment by andygnewman** [comment]
Nice work on the tests. 👍

**Pull request merged**